packages/react-components/src/localization/locales/de-DE/strings.json

```diff
@@ -15,12 +15,17 @@
   },
   "sendBox": {
     "placeholderText": "Nachricht eingeben",
-    "textTooLong": "Ihre Nachrichtenlänge überschreitet den maximalen Grenzwert."
+    "textTooLong": "Ihre Nachrichtenlänge überschreitet den maximalen Grenzwert.",
+    "sendButtonAriaLabel": "Nachricht senden"

```

Comment:
You don't need to update any of the translated strings. Just the en-US ones. The rest will be updated via autmation / td build.

Reply:
I just ran the td build to update the strings to make sure that the logic I wrote was correct.